3rdparty/CMakeLists.txt

```diff
@@ -1270,6 +1328,7 @@ EXTERNAL(protobuf ${PROTOBUF_VERSION} ${CMAKE_CURRENT_BINARY_DIR})
 
 # TODO(andschwa): Move protobufs so these don't have to be GLOBAL.
 add_library(protobuf ${LIBRARY_LINKAGE} IMPORTED GLOBAL)
+add_library(libprotoc ${LIBRARY_LINKAGE} IMPORTED GLOBAL)
```

Comment:
Yes, `protoc` is compiled with shared libraries, thus that library is necessary for runtime. I can wrap it with `if` since it is necessary only for module development (let me know).

Note, on my local environment I cannot reproduce that issue, I faced it building DCOS `mesos-module` package.